### PR TITLE
add more core samples in the sample test

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2030)

### CODE REVIEW

File: test/sample_test.yaml [hunk 1]
```diff
@@ -71,6 +71,10 @@ spec:
               - xgboost_training_cm
               - lightweight_component
               - dsl_static_type_checking
+              - pipeline_transformers
+              - secret
+              - sidecar
+              - volume_ops
```
Comment (elikatsis) [inline test/sample_test.yaml]: This line should be removed as well.

File: samples/core/volume_ops/volume_ops.py [hunk 2]
```diff
@@ -20,11 +20,11 @@
     name="VolumeOp Basic",
     description="A Basic Example on VolumeOp Usage."
 )
-def volumeop_basic(size):
+def volumeop_basic(size='10M'):
     vop = dsl.VolumeOp(
         name="create_pvc",
         resource_name="my-pvc",
-        modes=dsl.VOLUME_MODE_RWM,
+        modes=dsl.VOLUME_MODE_RWO,
```
Comment (elikatsis) [inline samples/core/volume_ops/volume_ops.py]: I'd say revert the changes in this file.

Comment (gaoning777) [reply samples/core/volume_ops/volume_ops.py]: Any concerns of making this change? I want to make sure these samples are covered with the tests.

Comment (gaoning777) [reply samples/core/volume_ops/volume_ops.py]: I just saw that it is included in the e2e tests. Any particular reason of placing the tests in e2e instead of the sample test?

File: test/sample_test.yaml (hunk 1 above)
Comment (gaoning777) [reply test/sample_test.yaml]: Done

File: samples/core/volume_ops/volume_ops.py (hunk 2 above)
Comment (gaoning777) [reply samples/core/volume_ops/volume_ops.py]: Done.

Comment (elikatsis) [reply samples/core/volume_ops/volume_ops.py]: >Any particular reason of placing the tests in e2e instead of the sample test?

Not really. I couldn't get what's the difference so I randomly chose to place it there.
If it shouldn't be there, feel free to move it.

Comment (gaoning777) [reply samples/core/volume_ops/volume_ops.py]: This is good. Thanks for adding it.
